torch/compiler/__init__.py

```diff
@@ -381,3 +383,30 @@ def is_dynamo_compiling() -> bool:
         >>>     # ...rest of the function...
     """
     return False
+
+
+@contextlib.contextmanager
+def skip_guard_eval_unsafe() -> Generator[None, None, None]:
```

Comment:
@jansel are you sure about not making it a stance? I really don't see why it shouldn't be a stance, you need to hit all the compilation first before you can actually go unsafe, this seems 100% what the stance API wants to capture.

Reply:
We could but, I think it is orthogonal to the existing stances.  You could have:
- skip_guard_eval_unsafe + error_on_recompile
- skip_guard_eval_unsafe + eager_on_recompile
- skip_guard_eval_unsafe + default

So this would need to be a different arg.

Reply:
Yup, sounds good to me! Separate arg.